src/platform/Linux/CHIPPlatformConfig.h

```diff
@@ -33,6 +33,8 @@
 using CHIP_CONFIG_PERSISTED_STORAGE_KEY_TYPE = const char *;
 #define CHIP_CONFIG_PERSISTED_STORAGE_MAX_KEY_LENGTH 16
 
+#define CHIP_CONFIG_LIFETIIME_PERSISTED_COUNTER_KEY "rotation-counter"
```

Comment:
Suggest shorter key and mention of lifetime (e.g. `life-count`)

Reply:
updated

Reply:
should this be the same as the other 'life-count' entry?